mm2src/mm2_main/src/rpc/lp_commands/lp_commands.rs

```diff
@@ -54,6 +56,7 @@ pub struct GetSharedDbIdResponse {
     shared_db_id: H160Json,
 }
 
+// TODO: Return shared_db_id for all available and active unique pubkeys
 pub async fn get_shared_db_id(ctx: MmArc, _req: Json) -> GetSharedDbIdResult<GetSharedDbIdResponse> {
```

Comment:
We probably don't need anything done here since the shared_db_id is unique and one across all coins, it's related to the seed we started mm2/kdf with.